woocommerce/woocommerce
# Also support syncing for HPOS with stats table.

### All Submissions:






### Changes proposed in this Pull Request:



Upgrade stats code to sync to HPOS as well as with posts.

Closes #35032.

### How to test the changes in this Pull Request:

1. Set HPOS as enabled and sync between posts and order table disabled.
2. Create a new order by going to the order admin screen or via the checkout. 
3. If order is created in this branch, you will see that it will be synced to the stats table (you may need to wait a few minutes for order sync action to run). On trunk, no sync action will be scheduled.

### Other information:




### FOR PR REVIEWER ONLY:
